File: source/kCura.Relativity.Export/Types/Pair.vb
Namespace  kCura.Relativity.Export.Types
	<Serializable()> Public Class Pair
		Public Value As String
		Public Display As String
		Public Overrides Function ToString() As String
			Return Me.Display
		End Function
		Public Sub New(ByVal v As String, ByVal d As String)
			Me.Value = v
			Me.Display = d
		End Sub
		Public Shadows Function equals(ByVal other As Pair) As Boolean
			Return (Me.Display = other.Display And Me.Value = other.Value)
		End Function
	End Class
End Namespace

File: source/kCura.Relativity.Export.ConvertedToC#/Types/Pair.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Data;
using System.Linq;
namespace kCura.Relativity.Export.Types
{
	[Serializable()]
	public class Pair
	{
		public string Value;
		public string Display;
		public override string ToString()
		{
			return this.Display;
		}
		public Pair(string v, string d)
		{
			this.Value = v;
			this.Display = d;
		}
		public new bool @equals(Pair other)
		{
			return (this.Display == other.Display & this.Value == other.Value);
		}
	}
}
